bedrock/firefox/redirects.py

```diff
@@ -515,4 +515,10 @@ def firefox_channel(*args, **kwargs):
     # Bug 1277196
     redirect(r'^firefox(?:\/\d+\.\d+(?:\.\d+)?(?:a\d+)?)?/firstrun/learnmore/?$', 'firefox.features'),
     redirect(r'^firefox/windows-10/welcome/?$', 'https://support.mozilla.org/kb/how-change-your-default-browser-windows-10'),
+
+    # Bug 1352139
+    # Only en-US visitors get new /firefox/ page (for now...)
+    # This does not work...
+    # PMAC: halp pls
+    redirect(r'^((?!en\-US))\w{2,3}(?:-\w{2})?/firefox/?$', '{locale}/firefox/new/'),
```

Comment:
Which you've already done. The redirect you're doing in the view means we don't need this.